collector/iis.go

```diff
@@ -991,7 +991,7 @@ var ApplicationStates = map[uint32]string{
 }
 
 // W3SVCW3WPCounterProvider_W3SVCW3WP returns names prefixed with pid
-var workerProcessNameExtractor = regexp.MustCompile(`^\d+_(.+)$`)
+var workerProcessNameExtractor = regexp.MustCompile(`^(\d+)$_(.+)$`)
```

Comment:
This looks strange - the regex seems like it couldn't work? There are two "end of string" (`$`), so it shouldn't ever match

Reply:
A tip about regex: Check out https://regex101.com/, in the left menu select "Golang" flavor.